snippets/gift-card-recipient-form.liquid

```diff
@@ -23,6 +23,8 @@
       name="{{ gift_card_recipient_control_flag }}"
       disabled
       unchecked
+      aria-expanded="false"
```

Comment:
I think the reason why this isn't working as you expected it is because it's missing a role. aria-expanded works with the following [associated roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#associated_roles). A button, for example, has an implicit role, so you wouldn't need to add one. But for an element without an implicit role, such as a div or this input, the screen reader will require more context to understand what the widget does.

I'm not super familiar with this area, but I'd be happy to take a look at it myself if you add some detailed testing steps (some simple instructions on how to get to this screen/checkbox would be helpful). Feel free to reach out if you'd like to pair too.

Reply:
Thanks, @ludoboludo. @antiBaconMachine and I have discussed this, as well as potential solutions. Here's a summary:
- we both agreed that a checkbox might not be the best element to use here
- I suggested changing the element to a `button` and using one of the following patterns: an [accordion](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/) or a [disclosure](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/) (which we use in the drawer menu)
- I suggested checking with UX to find a good pattern and adjust the design if needed

Reply:
Sweet, thanks for the update 🙂